composer.json

```diff
@@ -23,6 +23,7 @@
     "symfony/yaml": "~2.7",
     "embed/embed": "^2.6",
     "swiftmailer/swiftmailer": "~5.4",
+    "symfony/cache": "3.3.x-dev",
```

Comment:
Can we change to `^3.3@dev` which will still match the dev branch, but also allows stable to be installed once a tag is available.